application/platform/types/observe/origin/file.ts

```diff
@@ -10,6 +10,10 @@ import * as Types from '../types';
 import * as Parser from '../parser';
 import * as Sde from '../sde';
 import * as str from '../../../env/str';
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import * as tslib from '../../../../apps/rustcore/ts-bindings/node_modules/tslib';
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
```

Comment:
Why we need this? `tslib` isn't used directly in the code, it's dev-dependency

Reply:
Without this, we were getting the below errors.

![image](https://github.com/esrlabs/chipmunk/assets/17827621/f06cdae1-d42a-4ee2-9b4b-1390285809a0)

![image](https://github.com/esrlabs/chipmunk/assets/17827621/68a79584-a870-4c4a-9d36-1177d4dc71ee)